README.md

```diff
@@ -74,6 +74,8 @@ manually if your distribution does not provide a package.
 
 First, make sure you have the required __dependencies__:
 - build:
+	- autopoint (for bootstrapping)
+	- autoreconf (for bootstrapping)
```

Comment:
On some distros, this dep is called [sys-devel/autoconf-wrapper](https://github.com/gentoo/gentoo/tree/master/sys-devel/autoconf-wrapper).